runeq/resources/project.py

```diff
@@ -0,0 +1,182 @@
+"""
+Fetch metadata about projects.
+
+"""
+
+from typing import Iterable, Optional, Type
+
+from .client import GraphClient, global_graph_client
+from .common import ItemBase, ItemSet
+
+
+class Project(ItemBase):
+    """
+    Metadata for an project.
+
+    """
+
+    def __init__(
+        self,
+        id: str,
+        title: str,
+        status: str,
+        type: str,
+        started_at: float,
+        updated_at: float,
+        created_at: float,
+        created_by: str,
+        updated_by: str,
+        **attributes
+    ):
+        """
+        Initialize with data.
+
+        Args:
+            id: ID of the project
+            title: Human-readable name
+            created_at: When the project was created (unix timestamp)
+            status: Status of the project,
+            type: Type of project,
+            started_at: Time the project started (unix timestamp),
+            created_at: Time the project was created (unix timestamp),
+            created_by: UserID of who created the project,
```

Comment:
Perhaps add an example showing that the `user-` prefix is expected?

Reply:
Hmm actually after running the query locally it seems that `created_by` and `updated_by` are actually just user names not ids? For project with id `3ac2b0fe-0a6f-4195-9489-2f700be09ec9`, both `creates_by` and `updated_by` seem to be `"Nicholas"`. Can you double check that this is true and not just an exception and update the code to reflect that.

Reply:
That is accurate! I just miss understood what was being returned.